COMMUNICATIONS.md

```diff
@@ -0,0 +1,94 @@
+
+### Getting Started
+
+_Join Us_
+
+  * Register your account at https://opensearch.slack.com. You will be required
+    to enroll in two-factor authentication to ensure the best possible security
+    of your account.
+
+_Update Profile_
+
+  * You may add your interests or keywords in the “What I do” section of your 
+    profile along with your Title and Company/Organization. This helps 
+    participants network and connect with each other by common interests. No 
+    anonymous display names are allowed.
+  * Feel free to update your profile photo to something unique to you. All 
+    profile photos are subject to the 
+    [OpenSearch Code of Conduct](CODE_OF_CONDUCT.md) policies.
+  * Optionally share your pronouns by adding them to the end of your Full Name 
+    in your profile. This way other participants will be able to see them and 
+    be respectful to your communication preferences. Please also be respectful 
+    of other participants’ pronouns.
+
+### Workspace Channels
+
+After you join OpenSearch workspace you will be automatically added to the 
+following channels:
+
+  * **#general** - a channel for asking general questions. Based on the question 
+    or discussion users will mostly likely be directed to more specific 
+    channels based on the topics of interest.
+  * **#random** - consider this the water cooler in a face to face workspace. 
+    Feel free to introduce yourself to the community, share what you’re 
+    working on, or what you’re excited about learning with OpenSearch.
+
+There are dedicated channels for the following (note that participants might 
+be directed to more specific channels based on the nature of the discussion):
+
+  * **#dev** - primary channel for development questions. This is a good place to 
+    start for participants interested in OpenSearch development but are not 
+    sure which channels to join.  
+  * **#core** -  primary channel for core OpenSearch. General questions and 
+    discussions about the core can be asked here. 
+  * **#dashboards** - primary channel for Dashboards. General questions and 
+    discussions about dashboards can be asked here. 
+  * **#infra** - primary channel for infra. General discussions, questions, or infra 
+    requests can be posted here.
+  * **#{project}-build** - dedicated channels where build bots will post failing CI/CD 
+    builds for test failure triage and bug fixes.
+  * **#maintainers** - a private maintainers channel to discuss new maintainers, and 
+    project sensitive topics.
+
+### Tips
+
+  * Try to keep communications in the open - Not only does this facilitate better
+    sharing of information but back channeled discussions are not retained for
+    search among other community members.
+  * Use the [OpenSearch Discussion Forum](https://forum.opensearch.org) for 
+    technical support discussions or, at least, open a summary discussion on the 
+    forum so useful findings can be shared with the rest of the community. 
```

Comment:
I feel like this is weird. Use whatever you want. Generally I don't love recommendations of what "not" to do, which is "don't use slack for technical support discussions".

Should this doc have a section on forums and their purpose as part of this PR?

Reply:
> Should this doc have a section on forums and their purpose as part of this PR?

I don't think so; that can be addressed in a separate PR.


> Generally I don't love recommendations of what "not" to do, which is "don't use slack for technical support discussions".

I agree with this; the one wrinkle here is that the slack conversations are not searchable outside of the slack workspace. So if someone uses slack for technical support it will not be discoverable by the rest of the community. We can encourage folks to use the forum but if that's not documented somewher (like this "tips" section) it risks never being followed (just like forgetting to use slack threads to avoid the channel noise). 

For now I reworded to explicitly mention this tip and encourage using the forum instead of making it sound like a policy.

Reply:
The community will likely see the forum recommendation as a good thing - when given direction of where / what to use, most will appreciate it.